src/main/java/com/specialwarriors/conal/common/auth/jwt/JwtProvider.java

```diff
@@ -0,0 +1,81 @@
+package com.specialwarriors.conal.common.auth.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.Jwts.SIG;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class JwtProvider {
+
+    private final SecretKey secretKey;
+    @Value("${spring.jwt.access-token-exp}")
+    private long accessTokenExp;
+    @Value("${spring.jwt.refresh-token-exp}")
+    private long refreshTokenExp;
```

Comment:
```suggestion
    
    @Value("${spring.jwt.access-token-exp}")
    private long accessTokenExpiration;
    
    @Value("${spring.jwt.refresh-token-exp}")
    private long refreshTokenExpiration;
```
필드에 어노테이션을 사용할 경우 빈 라인을 두는 것이 가독성 측면에서 낫다고 생각하는데 붙여서 작성하신 이유, 의견 부탁 드립니다.

더불어 `Exp`로 `expiration`을 축약해 사용할 경우 의미 전달이 불명확해질 수 있다고 생각하는데, 약어를 사용하신 이유가 궁금합니다. 또한, 만료 기간을 밀리초 단위로 설정하신 것으로 보이는데 이 단위를 표현할 수 있도록 변수명을 짓는 것은 어떨까요? 검토 후 답변 부탁 드립니다.

Reply:
저도 필드 사이에 빈 라인을 두는걸 선호하는데 지나쳐버린 것 같습니다
exp로 줄인 이유는 expiration이 길기도 하고, jwt에서 access-token-exp, refresh-token-exp 두 개 모두 존재하므로 혼동할 여지가 없다고 판단해서 exp로 명명했습니다.
만료기간을 ms 말고 다른 단위로도 설정할 수 있나요..? 잘 몰라서 여쭤봅니다..!

Reply:
다른 단위로 설정하라는 맥락에서 드린 얘기는 아닙니다. 코드를 봤을때 `accessTokenExpirationMillis`와 같이 어떤 단위로 값을 표현한 것인지 알 수 있게 명명하는 것이 좋지 않냐는 의견이었습니다. 이름이 조금 길어지더라도 의미를 명확하게 하는 것이 보통 더 바람직하게 여겨지니 한 번 검토해주세요